README.markdown

```diff
@@ -6,6 +6,42 @@ standards such as [JSON Schema](http://json-schema.org), [JSON
 Pointer](https://www.rfc-editor.org/rfc/rfc6901),
 [JSONL](https://jsonlines.org), and more.
 
+Example
+
+```cpp
+#include <sourcemeta/jsontoolkit/json.h>
+#include <sourcemeta/jsontoolkit/jsonpointer.h>
+#include <sstream>
+#include <iostream>
+
+int main()  {
+    // Creating JSON using parse
+    sourcemeta::jsontoolkit::JSON document =
+        sourcemeta::jsontoolkit::parse(R"JSON({
+            "name": "John Doe",
+            "age": 20,
+            "address": "zxy"
+        })JSON");
+
+    // JSON pointer for the name property
+    const sourcemeta::jsontoolkit::Pointer name_pointer{"name"};
+    // A new JSON document for the name
+    const sourcemeta::jsontoolkit::JSON name_value{"Johnny Doe"};
+    // Updating the value of the name using JSON pointer
+    sourcemeta::jsontoolkit::set(document, name_pointer, name_value);
+    sourcemeta::jsontoolkit::prettify(document, std::cout);
+    std::cout<<"\n";
+
+    // The above program will print the following to standard output:
```

Comment:
Now that I see this again, I wonder if you should do this AFTER the code block. i.e. after showing the C++ program, write on the README: "The above program will print the following to standard output:", then open a new JSON code block, and put the output?